src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -656,4 +666,110 @@ void GET_cases_givenNoCreatedFilterParams_whenGetCases_thenReturnAllCases() {
         }
     }
 
+    @Nested
+    class OffenderEndpointTests {
```

Comment:
You could consider breaking these out into a separate class. We started using Nested classes a while back but honestly I'm not sure it was a good idea, it just produces massive tests. Interested to hear what your thoughts are?

Reply:
moved as suggested